src/sass/modules/_m-vet-nav.scss

```diff
@@ -36,29 +35,30 @@ body.va-pos-fixed {
     padding: 0 1.6rem;
     white-space: normal;
   }
+
+  &.vetnav--submenu-expanded {
+    #vetnav-menu {
+      overflow: hidden;
+      height: 0;
+    }
+  }
 }
 
 #vetnav-menu {
   height: auto;
   list-style: none;
-  margin: 1rem 0;
-  overflow-y: auto;
+  margin: 0;
```

Comment:
Is removing the top margin on mobile necessary? I personally think it looks better to balance out the spacing above and below the close button.

Reply:
The `$height-offset` has been increased to compensate for the margins being dropped so it should look the same. The margins were causing some of the content to be slightly cut off when scrolling on mobile devices because of the way height is calculated. We can increase that `$height-offset` though if you want to balance it out more.

Reply:
It's fine. Testing on the menu before the change indicates that the spacing under the button shouldn't be there anyway.